src/components/navbar/Navbar.module.scss

```diff
@@ -0,0 +1,22 @@
+.navbar {
+  background-color: $color-white;
+  color: $color-primary;
+  display: flex;
+  justify-content: space-between;
+  padding: $space-5 $space-200;
+
+  &__logo {
+    display: flex;
+    align-items: center;
+
+    &--img {
+      padding-right: $space-10;
+      width: $space-30;
+    }
+
+    &--text {
+      font-size: 30px;
```

Comment:
Use variables. 

Or usually we introduce font related variables like spacing variables.

For example variables for weights and so on.